advanced_functionality/working_with_tfrecords/tfrecord-transformer-container/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+set -e
+
+# Get the listen port from the SM env variable, otherwise default to 8080
+LISTEN_PORT=${SAGEMAKER_BIND_TO_PORT:-8080}
+
+# Start flask app
+exec flask run --host=0.0.0.0 --port=$LISTEN_PORT
```

Comment:
don't run flask http in production. instead, use gunicorn in front of flask. byo scikit has an easy-to-follow example of this.

Reply:
Updated to use gunicorn as the http server